docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -16,15 +16,15 @@ You can use the [TechPass portal](http://portal.techpass.gov.sg) to do a self-se
 
 ## Audience
 
-- Public officers using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name\>@\<acronym for your agency\>.gov.sg*.
+- Public officers using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name\>@\<acronym for your agency\>.gov.sg*. For example, *peter_wilson@tech.gov.sg*.
 
-- Vendors or contractors using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name_from\>.\<vendor organisation name\>@\<acronym for the agency\>.gov.sg*.
+- Vendors or contractors using a non-SE GSIB device and whose organisational email address is in the format of *\<your_name_from\>.\<vendor organisation name\>@\<acronym for the agency\>.gov.sg*. For example, *peter_wilson_FROM.vendorprovider@tech.gov.sg*.
```

Comment:
may I trouble you to please add a note for vendors.

from@ emails support are only for direct temporary hires of the agencies. all vendors are strongly encouraged to use their company email for sign up. This is to prevent issues of access controls and provisioning problems for agencies and products

Reply:
I have a small doubt here.

Reply:
Just curious , is our current info on https://docs.developer.tech.gov.sg/docs/techpass-user-guide/onboard-public-officers-using-non-se-machines not conveying this info and do we encounter issues of access controls and provisioning because of this?
i can add the above note but my understanding is only if a staff is deputed to agencies via vendor organisation, they have email address with "From." format.
Direct contractors (directly recruited by agencies ) will have the WOG email  address similar to PO.  so is from@ emails support are only for direct temporary hires of the agencies  is that correct?